pkgs/servers/zigbee2mqtt/default.nix

```diff
@@ -3,14 +3,15 @@ let
   package = (import ./node.nix { inherit pkgs system; }).package;
 in
 package.override rec {
-  version = "1.16.1";
+  version = "1.16.2";
   reconstructLock = true;
 
   postInstall = ''
     sed -i '1s;^;#!/usr/bin/env node\n;' $out/lib/node_modules/zigbee2mqtt/index.js
-    chmod +x $out/lib/node_modules/zigbee2mqtt/index.js
+    chmod +x $out/lib/node_modules/zigbee2mqtt/cli.js
+    rm -rf $out/bin
     mkdir $out/bin
-    ln -s $out/lib/node_modules/zigbee2mqtt/index.js $out/bin/zigbee2mqtt
+    ln -s $out/lib/node_modules/zigbee2mqtt/cli.js $out/bin/zigbee2mqtt
```

Comment:
any reason not to use upstream generated binary? that way the chmod, rm,mkdir and ln commands would be come obsolete.
or is upstream bin/zigbee2mqtt symlink not working afterwards?

Reply:
nit: s/binary/executable/

Reply:
Good point. I [removed the `postInstall` completely](https://github.com/NixOS/nixpkgs/commit/b1776410b3c7ccaa6a3476e1bc5ea87cb4312882) and the NixOS test is running just fine.